.eslintrc.js

```diff
@@ -5,11 +5,11 @@ module.exports = {
     '!.eslintrc.js',
     '!jest.config.js',
     'node_modules',
-    'dist',
-    'docs',
-    'coverage',
+    '**/dist',
+    '**/docs',
+    '**/coverage',
```

Comment:
> Glob patterns in ignorePatterns are relative to the directory that the config file is placed in.
(https://eslint.org/docs/latest/use/configure/ignore#ignorepatterns-in-config-files)

The `.gitignore` file lists these as:
```
packages/*/coverage
packages/*/dist
packages/*/docs
scripts/coverage
```

Reply:
Thanks! That makes more sense.